FalmouthFalcons/iteration-2
# customer tests all run without errors!!!

Description:
_write and run tests for CRUD of customers to db

Number of Fixes:
_all new code, no fixes 

Related Ticket(s):
_Issue ticket #1  

Problem to Solve:
_no problems 

Proposed Changes:
_no changes 

Expected Behavior:
_all tests passed re: customer.rb and customer_test.rb

Steps to Test Solution
1)git pull origin master 
2)git fetch 
3)git checkout branch "adam-issue-1"

Documentation
[NO ] There is new documentation in this pull request that must be reviewed..
[ NO ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
